chainer/training/extensions/plot_report.py

```diff
@@ -23,28 +24,28 @@
 
 def _check_available():
     if not _available:
-        msg = '''matplotlib is not installed on your environment.
-Please install matplotlib to plot figure.
-
-  $ pip install matplotlib'''
-        raise RuntimeError(msg)
+        warnings.warn('matplotlib is not installed on your environment, '
+                      'so that nothing will be plotted at this time. '
+                      'Please install matplotlib to plot figures.\n\n'
+                      '$ pip install matplotlib')
 
 
 class PlotReport(extension.Extension):
+
     """Trainer extension to output plots.
 
     This extension accumulates the observations of the trainer to
-    :class:`~chainer.DictSummary` at a regular interval specified by a
```

Comment:
Basically yes, but these lines have unnecessary spaces before changing lines.
So I removed those spaces to shrink the number of lines, but resulted in the same number of lines.